app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewScreen.kt

```diff
@@ -0,0 +1,146 @@
+package com.kickstarter.features.pledgedprojectsoverview.ui
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListState
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.kickstarter.R
+import com.kickstarter.ui.compose.designsystem.KSTheme
+import com.kickstarter.ui.compose.designsystem.KSTheme.colors
+import com.kickstarter.ui.compose.designsystem.KSTheme.dimensions
+import com.kickstarter.ui.compose.designsystem.KSTheme.typography
+import com.kickstarter.ui.toolbars.compose.TopToolBar
+
+@Composable
+@Preview(name = "Light", uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(name = "Dark", uiMode = Configuration.UI_MODE_NIGHT_YES)
+private fun PledgedProjectsOverviewScreenPreview() {
+    KSTheme {
+        Scaffold(
+            backgroundColor = colors.backgroundAccentGraySubtle
+        ) { padding ->
+            PledgedProjectsOverviewScreen(
+                modifier = Modifier.padding(padding),
+                lazyColumnListState = rememberLazyListState(),
+                ppoCards = (0..10).map {
+                    PPOCardDataMock()
+                },
+                alertsAmount = 10,
+                onBackPressed = {}
+            )
+        }
+    }
+}
+
+@Composable
+fun PledgedProjectsOverviewScreen(
+    modifier: Modifier,
+    alertsAmount: Int = 0,
+    onBackPressed: () -> Unit,
+    lazyColumnListState: LazyListState,
+    ppoCards: List<PPOCardDataMock> = listOf(),
```

Comment:
fyi, In case you've decided using pagin3, you could directly use here LazyPagingItems<PPOCardDataMock>

Reply:
Cool, will update this